### PR TITLE
feat(route): add 三立新聞網新聞總覽

### LINKED ISSUE #12180: 三立新聞網請求訂閱
### 类型

传统媒体

### 网站地址

https://www.setn.com/ViewAll.aspx

### 网站描述

台灣新聞https://www.setn.com/ViewAll.aspx

### 需要生成什么内容？

訂閱即時新聞https://www.setn.com/ViewAll.aspx

### 额外描述

https://www.setn.com/ViewAll.aspx

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #12180

## 完整路由地址 / Example for the proposed route(s)

```routes
/setn/即時
/setn/熱門
/setn/娛樂
/setn/政治
/setn/社會
/setn/國際
/setn/兩岸
/setn/生活
/setn/健康
/setn/旅遊
/setn/運動
/setn/地方
/setn/財經
/setn/富房網
/setn/名家
/setn/新奇
/setn/科技
/setn/汽車
/setn/寵物
/setn/女孩
/setn/HOT焦點
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/setn/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const a = item.find('a').last();
+            const link = a.attr('href');
+
+            return {
+                title: a.text(),
+                link: /^http/.test(link) ? link : `${rootUrls.hasOwnProperty(category) ? rootUrls[category] : rootUrl}${link}`,
```
Comment (TonyRL) [inline lib/v2/setn/index.js:59]: Please remove the tracking parameter `utm_campaign`

File: lib/v2/setn/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    url: item.link,
+                });
+
+                const content = cheerio.load(detailResponse.data);
+
+                content('#gad_setn_innity_oop_1x1').remove();
+
+                item.author = content('meta[property="author"]').attr('content');
+                item.category = [content('meta[property="article:section"]').attr('content'), ...content('meta[name="news_keywords"]').attr('content').split(',')];
+                item.pubDate = parseDate(content('meta[property="article:published_time"]').attr('content'));
```
Comment (TonyRL) [inline lib/v2/setn/index.js:77]: timezone adjustment required

File: docs/traditional-media.md [hunk 3]
```diff
@@ -1986,6 +1986,24 @@ category 对应的关键词有
...
+
+| 國際 | 兩岸 | 生活 | 健康 | 旅遊 |
+| ---- | ---- | ---- | ---- | ---- |
+
+| 運動 | 地方 | 財經 | 富房網 | 名家 |
+| ---- | ---- | ---- | ------ | ---- |
+
+| 新奇 | 科技 | 汽車 | 寵物 | 女孩 | HOT 焦點 |
+| ---- | ---- | ---- | ---- | ---- | -------- |
+
```
Comment (TonyRL) [inline docs/traditional-media.md:2006]:
```suggestion

</Route>

```

File: lib/v2/setn/index.js (hunk 1 above)
Comment (nczitzk) [reply lib/v2/setn/index.js:59]: 👌

**Pull request merged**